contracts/colonyNetwork/ColonyNetworkSkills.sol

```diff
@@ -146,7 +151,9 @@ contract ColonyNetworkSkills is ColonyNetworkStorage, Multicall {
     // This succeeds if not set, but we don't want to block e.g. domain creation if that's the situation we're in,
     // and we can re-call this function to bridge later if necessary.
     (bool success, ) = miningBridgeAddress.call(payload);
-    require(success, "colony-network-unable-to-bridge-skill-creation");
+    if (!success) {
+      emit SkillCreationStored(_skillId);
```

Comment:
Is anything actually stored here?

Reply:
The name of the event was to be consistent with other events related to bridging where things are explicitly stored in the event of failure.

Reattempts are handled by calling this function again, as the comments indicate.